src/components/calcite-action-group/resources.ts

```diff
@@ -1,3 +1,8 @@
 export const SLOTS = {
-  menuActions: "menu-actions"
+  menuActions: "menu-actions",
+  menuTooltip: "menu-tooltip"
+};
+
+export const TEXT = {
```

Comment:
Is this used in another PR? If not, this can be dropped.

Reply:
Yes it is.